week-3/F-array-forEach/exercise.js

```diff
@@ -6,9 +6,47 @@
 
   An array with numbers 0-15 has been provided.
 */
+//create a function that performs on each element in the array.
 
 var arr = [0, 1, 2, 3, 4, 5, 6, 7, 8, 9, 10, 11, 12, 13, 14, 15];
 
+arr.forEach(multiples(item){ 
```

Comment:
I think you're missing an important keyword on this line (line 13) :) Look at your E-array-map/exercise.js to help you figure out what it is.